feathr_project/test/test_feature_materialization.py

```diff
@@ -57,3 +65,48 @@ def test_feature_materialization_now_schedule():
     assert expected.year == date.year
     assert expected.month == date.month
     assert expected.day == date.day
+
+def test_feature_materialization_now_schedule():
+    """Test back fill cutoff time without backfill."""
+    settings = MaterializationSettings("", [], [])
+    date = settings.get_backfill_cutoff_time()[0]
+    expected = datetime.now()
+    assert expected.year == date.year
+    assert expected.month == date.month
+    assert expected.day == date.day
```

Comment:
what's this test for? Is this related to this PR? I am a bit out of context for this change.

Reply:
Oops, sorry. My mistake. Copy-paste error.